usr/bin/torbrowser

```diff
@@ -346,7 +362,7 @@ maybe_use_open_link_confirmation() {
       ## maybe use open link confirmation
       if [ -x /usr/lib/open_link_confirmation ]; then
          if [ ! "$open_link_confirmation_skip" = "true" ]; then
-            tool=/usr/lib/open_link_confirmation
+            tool="/usr/lib/open_link_confirmation $tb_using_i2p"
```

Comment:
I guess `$tb_using_i2p`  would fit better below where `$tool` gets executed?

Reply:
The reason that's there is so I could tell /usr/lib/open_link_confirmation what variant browser it's opening. WIthout it(and the changes in my fork of open-link-confirmation) the dialog created by open_link_confirmation reports Tor Browser instead, but opens the i2p profile from ~/.i2pb/. I missed this when I switched over to environment variables with the revision. It can just be removed, and I'll make my patch to open_link_confirmation use the environment variables instead.